src/bp-settings/classes/class-bp-rest-account-settings-options-endpoint.php

```diff
@@ -740,7 +740,7 @@ public function get_notifications_fields() {
 
 			$fields_forums[] = array(
 				'name'        => 'notification_forums_following_reply',
-				'label'       => __( 'A member replies to a discussion you are subscribed', 'buddyboss' ),
+				'label'       => __( 'A member replies to a discussion you are subscribed to', 'buddyboss' ),
```

Comment:
esc_html__( 'A member replies to a discussion you are subscribed to', 'buddyboss' ),